tests/onnxruntime/test_optimization.py

```diff
@@ -55,26 +56,27 @@ def test_save_and_load(self):
 class ORTOptimizerTest(unittest.TestCase):
 
     SUPPORTED_ARCHITECTURES_WITH_MODEL_ID = (
-        (BertForSequenceClassification, "hf-internal-testing/tiny-random-bert"),
-        (DistilBertForSequenceClassification, "hf-internal-testing/tiny-random-distilbert"),
-        (BartForSequenceClassification, "hf-internal-testing/tiny-random-bart"),
-        (GPT2ForSequenceClassification, "hf-internal-testing/tiny-random-gpt2"),
-        (RobertaForSequenceClassification, "hf-internal-testing/tiny-random-roberta"),
-        (ElectraForSequenceClassification, "hf-internal-testing/tiny-random-electra"),
-        (XLMRobertaForSequenceClassification, "hf-internal-testing/tiny-xlm-roberta"),
-        (BigBirdForSequenceClassification, "hf-internal-testing/tiny-random-big_bird"),
+        (BartForSequenceClassification, "hf-internal-testing/tiny-random-bart", "sequence-classification"),
+        (BertForSequenceClassification, "hf-internal-testing/tiny-random-bert", "sequence-classification"),
+        (BigBirdForSequenceClassification, "hf-internal-testing/tiny-random-big_bird", "sequence-classification"),
+        (DistilBertForSequenceClassification, "hf-internal-testing/tiny-random-distilbert", "sequence-classification"),
+        (ElectraForSequenceClassification, "hf-internal-testing/tiny-random-electra", "sequence-classification"),
+        (GPT2ForSequenceClassification, "hf-internal-testing/tiny-random-gpt2", "sequence-classification"),
+        (MT5ForConditionalGeneration, "hf-internal-testing/tiny-random-mt5", "seq2seq-lm"),
+        (RobertaForSequenceClassification, "hf-internal-testing/tiny-random-roberta", "sequence-classification"),
+        (XLMRobertaForSequenceClassification, "hf-internal-testing/tiny-xlm-roberta", "sequence-classification"),
```

Comment:
I added the `sequence-classification` feature for others and `seq2seq-lm` for the MT5 model.